### DISCUSSION TITLE
Can't upload PDF files

### DISCUSSION THREAD

**Discussion by marcguasch1** [AllYourBot/hostedgpt#522] [General]
Can someone please help? I am unable to upload PDF files for quote mining. I am not sure what I have done wrong as I set up according to the instructions.

<img width="604" alt="Screenshot 2024-10-17 151741" src="https://github.com/user-attachments/assets/28450705-2c5f-4642-86bb-7fcefb07b17b">
<img width="738" alt="Screenshot 2024-10-17 151727" src="https://github.com/user-attachments/assets/82018967-8207-424f-a2ea-c1bcc6e4dd6b">

**Comment by keithschacht** [comment]
@marcguasch1 The app currently only supports image uploads. Images can be processed natively by LLMs so that was fairly easy to implement, but uploading other kinds of docs requires a bit more steps. They need to be pre-processed and stored in a special way in order for LLMs to be able to answer questions about them. I need to find someone to help with this implementation before the app can have support for it.
